### DISCUSSION TITLE
how to simplify the long path "_@tinacms_schema-tools@0.0.3@@tinacms/schema-tools/dist/types/SchemaTypes"

### DISCUSSION THREAD

**Discussion by scil** [tinacms/tinacms#2909] [Help] [answered]
my collections in schema.ts are long ,so I split them into seperate files like  this `page.ts`  

```
import { TinaCloudCollection } from "_@tinacms_schema-tools@0.0.3@@tinacms/schema-tools/dist/types/SchemaTypes";

export let pageCollection:TinaCloudCollection<false>  = {
  label: "Pages",
  name: "page",
  path: "content/pages",
  fields: [ ]
}

```

I think the path of the type `"_@tinacms_schema-tools@0.0.3@@tinacms/schema-tools/dist/types/SchemaTypes";`  is complicated,  could I have any way to simplify it?

**Comment by jamespohalloran** [comment, marked as answer]
I believe `TinaCloudCollection` is also exported from the `tinacms` package. 
```
     import { TinaCloudCollection } from "tinacms";
```

**Reply by scil** [reply]
Thank you. I wonder why JetBrains WebStorm only found the long path, not the simple one.
